modules/ROOT/pages/kafka/kafka-connector-reference.adoc

```diff
@@ -129,188 +135,182 @@ endif::[]
 | Keytab a| String |  Path to keytab file associated with "principal". |  | x
-| Service name a| String |  The Kerberos principal name that Kafka Broker runs as. |  | x
-| Additional JAAS Properties a| Object |  Additional parperties as key->value that you need to set on "sasl.jaas.config" and that you usually include in JAAS configuration file. |  |
+| Service Name a| String |  The Kerberos principal name that Kafka Broker runs as. |  | x
+| Additional JAAS Properties a| Object |  Additional properties such as a key and value that you need to set in `sasl.jaas.config` and that you usually include in the JAAS configuration file. |  | 
 |===
+
 [[kafka-producer-config_kerberos-ssl-kafka-producer-connection]]
 ===== Kafka Kerberos SSL Producer Connection
 
-
 ====== Parameters
-[cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
+[%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Bootstrap Servers a| String |  Comma-separated host-port pairs used for establishing the initial connection to the Kafka cluster. This is the same as the "bootstrap.servers" value you must provide to Kafka clients (producer/consumer). If this property is provided with producer/consumer properties files, this value is ignored and the one from the properties file is used. |  | x
-| Additional Properties a| Object |  Additional properties as key->value that you need for your connection. |  |
-| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment will fail if the test doesn't pass after exhausting the associated reconnection strategy |  |
-| Key Store Type a| String |  The file format of the key store file. This is optional for client. |  JKS |
-| Key Store Password a| String |  The store password for the key store file. This is optional for client and only needed if "keyStoreLocation" is configured. |  |
-| Key Store Location a| String |  The location of the key store file. This is optional for client and can be used for two-way authentication for client. |  |
-| Trust Store Type a| String |  The file format of the trust store file. |  JKS |
-| Trust Store Password a| String |  The password for the trust store file. If a password is not set access to the truststore is still available, but integrity checking is disabled. |  | x
+| Bootstrap Servers a| String |  Comma-separated host-port pairs used for establishing the initial connection to the Kafka cluster. The same as the `bootstrap.servers` value you must provide to Kafka clients (producer/consumer). |  | x
+| Additional Properties a| Object |  Additional properties such as a key and value that you need for your connection. |  | 
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
```

Comment:
```suggestion
| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
```
I'm going to stop marking these to cut down on comments. You can S/R and also S/R produce/consumer -> producer and consumer